assets/js/components/date-range-selector.js

```diff
@@ -111,9 +111,16 @@ class DateRangeSelector extends Component {
 				name="time_period"
 				label=""
 				onEnhancedChange={ this.handleSelection }
-				options={ options }
-				value={ dateValue }
-			/>
+				value={ `${ dateValue }` }
+			>
+				{ Object.keys( options ).map( ( option ) => {
+					return (
+						<Option key={ option } value={ option }>
+							{ options[ option ] }
+						</Option>
+					);
+				} ) }
```

Comment:
Do we need to do this here? Just double checking, since I kind of would expect the `options` prop of `Select` to be able to deal with this. Also I wanna make sure this doesn't cause an unexpected change in the functionality here (I doubt it would but this just stood out to me).

Reply:
The options prop only accepts an array, from what I can tell, so expects the values and labels to be the same. This surprised me too, as I would've expected an object with `{ [ value ]: label }` to work, but it doesn't 😞